<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HospitalizationTransferDialog</class>
 <widget class="QDialog" name="HospitalizationTransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Перенос госпитализации</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="CDateEdit" name="edtTransferDate">
     <property name="whatsThis">
      <string>дата окончания осмотра</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblComment">
     <property name="text">
      <string>Причина переноса:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblTransferDate">
     <property name="text">
      <string>Новая плановая дата госпитализации:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="edtComment"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="grpConsultation">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Консультация врача</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>475</width>
           <height>347</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtDiagnosis"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblRecommentedTreatment">
            <property name="text">
             <string>Рекомендации:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblDiagnosis">
            <property name="text">
             <string>Диагноз:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblTreatmentMethod">
            <property name="text">
             <string>Метод лечения:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTextEdit" name="edtTreatmentMethod"/>
          </item>
          <item row="5" column="0">
           <widget class="QTextEdit" name="edtRecommentedTreatment"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblTreatmentOrgStructure">
            <property name="text">
             <string>Лечебное отделение:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="edtTreatmentOrgStructure"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtComment</tabstop>
  <tabstop>edtTransferDate</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>edtDiagnosis</tabstop>
  <tabstop>edtTreatmentMethod</tabstop>
  <tabstop>edtRecommentedTreatment</tabstop>
  <tabstop>edtTreatmentOrgStructure</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HospitalizationTransferDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HospitalizationTransferDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
